src/pages/MapView.tsx

```
@@ -15,6 +15,23 @@
   const { state: searchValue, onChange, onClickReset } = useInput();
   const [isInputDisabled, setIsInputDisabled] = useState(false);
 
+  const location = useLocation();
+  const data = location.state?.data;
+
+  useEffect(() => {
+    if (data && data.places) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)